app/views/authors/index.html.erb

```diff
@@ -72,5 +72,5 @@
 </div>
 
 <% if user_signed_in? %>
-  <%= link_to "Add new author", new_author_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" %>
+  <%= link_to t('authors.add_new'), new_author_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" %>
```

Comment:
use this path:

```
en:
  authors:
    index:
      add_new: Add new author
```

then in the index view you can fetch the locale `t('.add_new')`

Reply:
https://github.com/yuliatokaryk/quotes/pull/24/commits/dc2b4fe4ba91acdf6a957323b45be257ea0fe351